data/fixtures/scopes/python/collectionItem.unenclosed2.scope

```diff
@@ -2,12 +2,12 @@ import foo, bar
 ---
 
 [#1 Content] =
-[#1 Domain] = 0:7-0:10
-         >---<
+[#1 Domain] = 0:0-0:10
```

Comment:
NB: #1 on the right side is new; the old 1 has become 2, the old 2 has become 3

The new one here is the extra scope generated by the textual source.

As above this won't actually change you behavior because when items are actually specifically requested we are going to end up the same list, `import ` won't be included